### ISSUE TITLE
Installation of 0.2.0 fails

### ISSUE THREAD

**Issue by iwarshak** [bratsche/twix#2]
Just added twix to my app and immediately got this error:

```
== Compilation error in file lib/twix/default_config.ex ==
** (Code.LoadError) could not load /Users/ian/work/project/app/deps/twix/config/config_utils.exs. Reason: enoent
    (elixir 1.14.0) lib/code.ex:1807: Code.find_file!/2
    (elixir 1.14.0) lib/code.ex:1184: Code.eval_file/2
    lib/twix/default_config.ex:2: (module)
```

**Comment by bratsche** [comment]
Hi @iwarshak, thanks for pointing this out! It seems to work fine when you set your mix.exs to point to the GitHub link directly, but it fails when it's set to 0.2.0 on hex.pm. I'm not sure why yet, but I'll investigate.

**Comment by bratsche** [comment]
Fixed in 0.3.0 release. Thank you @iwarshak! 🎉
